contrib/terraform/openstack/inventory.tf

```diff
@@ -0,0 +1,45 @@
+locals {
+  etcd_name = ["${concat(
```

Comment:
Some serious magic going on below here as well. Any chance comments can be added to clarify what's going on?